src/extra/plugin/remote.js

```diff
@@ -264,9 +264,8 @@ window.ParsleyConfig.validators.remote = {
   priority: -1
 };
 
-$(document).on('form:submit.parsley', function (evt, parsleyForm) {
-  if (!evt.isDefaultPrevented())
-    parsleyForm._remoteCache = {};
+window.Parsley.on('form:submit', function () {
+  this._remoteCache = {};
```

Comment:
Right, but with our own API, the way to "prevent the default" is to interrupt the events, there's no other way, in which case we're not even called.